### PR TITLE
LRQA-82562

### PR DESCRIPTION
**Pull request by crislucas1**
https://liferay.atlassian.net/browse/LRQA-82562

### CODE REVIEW

**Comment by crislucas1** [comment]
ci:test:sf

**Comment by crislucas1** [comment]
ci:test:sf

**Comment by crislucas1** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests/MyBlogTests.testcase [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		Navigator.gotoPage(pageName = "Blogs Page");
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests/MyBlogTests.testcase:31]: Hey @crislucas1, you need to put the `tearDown` to delete the blog and page that you created during your test

File: portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests/MyBlogTests.testcase [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+			BlogsNavigator.openToAddEntry(
+				layoutName = "Blogs Page",
+				siteURLKey = "guest");
+		}
+
+		task ("And a blog is created") {
+			BlogsEntry.addTitle(entryTitle = "Blogs Entry Title");
+
+			BlogsEntry.addContent(entryContent = "Blogs Entry Content");
+		}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests/MyBlogTests.testcase:46]: Hey @crislucas1, You have already created your blog with this title and the content you placed there. But you need to validate that the blog was actually created, so you need to go back to your Blogs Page and check if the blog with the title and content you entered is present on the page.

**Comment by crislucas1** [comment]
ci:test:sf

**Review by ViniLopes20 — APPROVED** [review]
Very good, @crislucas1. Congratulations, thanks a lot for working on it 🎉

**Pull request closed without merging**